doc/tutorial/essentials_quiz

```diff
@@ -0,0 +1,555 @@
+                                    <ol type="A">
+                                        <li><input type="radio" value="A" /><label for="A">fcm commit|ct|ci</label><br/></li>
+                                        <li><input type="radio" value="B" /><label for="B">fcm commit|ct</label><br/></li>
+                                        <li><input type="radio" value="C" /><label for="C">fcm commit|ci</label><br/></li>
+                                        <li><input type="radio" value="D" /><label for="D">fcm ct|ci</label><br/></li>
+                                    </ol>
+                                    <p class="answers C"><em>C</em></p>
+                                </div>
+                            </li>
+                            <li>
+                                <p class="question">What is the best command for finding out in what aspects your working copy is different from the repository, i.e. what has been modified etc?</p>
+                                <div id="5" class="questions">
+                                    <ol type="A">
+                                        <li><input type="radio" value="A" /><label for="A">fcm update</label><br/></li>
+                                        <li><input type="radio" value="B" /><label for="B">fcm trunk</label><br/></li>
+                                        <li><input type="radio" value="C" /><label for="C">fcm status</label><br/></li>
+                                        <li><input type="radio" value="D" /><label for="D">fcm info</label><br/></li>
+                                    </ol>
+                                    <p class="answers C"><em>C</em></p>
+                                </div>
+                            </li>
+                            <li>
+                                <p class="question">How would you find out the revision number of your working copy?</p>
+                                <div id="6" class="questions">
+                                    <ol type="A">
+                                        <li><input type="radio" value="A" /><label for="A">fcm status</label><br/></li>
+                                        <li><input type="radio" value="B" /><label for="B">fcm info</label><br/></li>
+                                        <li><input type="radio" value="C" /><label for="C">fcm revision</label><br/></li>
+                                        <li><input type="radio" value="D" /><label for="D">fcm query</label><br/></li>
+                                    </ol>
+                                    <p class="answers B"><em>B</em></p>
+                                </div>
+                            </li>
+                            <li>
+                                <p class="question">How would you find out if you are working on the trunk or a branch version and if a branch what it is called?</p>
+                                <div id="7" class="questions">
```

Comment:
Not sure I agree with your answer as being "D" based on the question as the revision number doesn't tell you the branch name.